lib/codit/population/networks/city_config/city_cfg.py

```diff
@@ -14,6 +14,9 @@ def read_demographic_data(section, file_name, data_dir=share_dir() / "codit/data
 DEMO_DATA = (read_demographic_data('population', 'table-8.csv')["%"] / 100).to_dict()
 ROUNDING = 0.001  # rounding such that sum of rates equals 1
 
+# Total population estimate (2019) Source: ONS* https://observatory.leeds.gov.uk/population/
+POPULATION_LEEDS = 793139
```

Comment:
don't want to hardcode anything about Leeds, as the code should be generic to any city

Reply:
Sure,  I can change the POPULATION_LEEDS  to POPULATION_CITY.

Reply:
Not sure I understand. Doesn't every city have a different population?

Reply:
POPULATION_LEEDS only defines the number of population in Leeds. It's not hardcoded, The population size of Outbreak is still controlled by POP_SIZE in SCALE_SETTINS dict defined in City_Overview.ipynb. I only passed this Macro POPULATION_LEEDS  to POP_SIZE in the City_Overview.ipynb to make the number easier to read.